ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-isis-new.md

```diff
@@ -156,9 +156,9 @@ interface Loopback1
 
 ##### ISIS
 
-| Interface | ISIS Instance | ISIS BFD | ISIS Metric | Mode |
-| --------- | ------------- | -------- | ----------- | ---- |
-| Vlan4093 | EVPN_UNDERLAY | - | 50 | point-to-point |
+| Interface | ISIS Instance | ISIS BFD | ISIS Metric | Mode | Level-1 Authentication Mode | Level-2 Authentication Mode |
```

Comment:
I think, we should create a separate table for `isis_authentication`
@MaheshGSLAB @ClausHolbechArista

Reply:
Any reason of having is separate as we may have again big table for same vlans just for the authentication.

Reply:
to cover all the parameters of authentication, or else we can add more columns in this table like keys and key-ids etc

Reply:
Is this much information of isis authentication is sufficient in table? 
@ClausHolbechArista

Reply:
For the column - `| Level-1 Authentication Mode | Level-2 Authentication Mode |`

we can give values like -
```
sha key-id 5
sha key-id 2 rx-disabled
shared-secret profile profile1 algorithm sha-1 rx-disabled
```

Reply:
To not overdo it, I think we should just have single `Authentication mode` column, and then add only the mode like `shared-secret` or `level1: shared-secret, level2: sha256` or whatever.

Reply:
Updated the ISIS table.